lib/app/features/escape_manual/data/model/escape_manual_remote.g.dart

```diff
@@ -108,3 +109,9 @@ Map<String, dynamic> _$EscapeManualTaskRemoteModelToJson(
       const JsonSecondsFromEpochConverter().toJson(instance.updatedAt));
   return val;
 }
+
+const _$EscapeManualTaskTypeEnumMap = {
```

Comment:
Não tem como definir esse enum uma única vez e usar em todos os lugares que precisamos?
Porque estamos definindo ele tanto aqui no `escape_manual_remote` quanto no `escape_manual_local`, e isso pode abrir espaço para inconsistências.

Reply:
Somente se unificar esses dois arquivos em um, pois esse arquivo é gerado pelo `build_runner`, como isso é automático ambos sempre serão atualizados caso o enum for modificado.

Reply:
Ah... esse é gerado de forma automática? não tinha percebido isso!

Então blz :)